app/views/hackdays/_project_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3> add new project </h3>
+	<div class="row"> 
+		<%= form_for(@new_project) do |f| %>
```

Comment:
(You're using tabs instead of spaces. Prefer 2 spaces for indentation in Ruby and JS)